src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -640,7 +640,7 @@
     <PropertyGroup>
       <CMakeDefinitionSaveFile>$(IntermediateOutputPath)..\cmake.definitions</CMakeDefinitionSaveFile>
     </PropertyGroup>
-    <Exec Command="python $(MSBuildThisFileDirectory)..\scripts\check-definitions.py &quot;$(CMakeDefinitionSaveFile)&quot; &quot;$(DefineConstants)&quot; &quot;$(IgnoreDefineConstants)&quot; " />
+    <Exec Command="&quot;$(PYTHON)&quot; $(MSBuildThisFileDirectory)..\scripts\check-definitions.py &quot;$(CMakeDefinitionSaveFile)&quot; &quot;$(DefineConstants)&quot; &quot;$(IgnoreDefineConstants)&quot; " />
```

Comment:
@silvioguiso this is the line that broke stuff.  Now that python is an env var and not a literal you need to do this everywhere else it gets called, OR make it conditional based using MSbuild conditions.

Reply:
Yikes. Sorry about that.

Should I revert the entire patch or should I go for targeted fixes?

If you want me to go for more targeted fixes, is there some way I can access logs that show where this PR breaks things?

Reply:
We fixed this in this PR, no need to revert.
https://github.com/dotnet/coreclr/pull/28102